### PR TITLE
PMM-9992 reverse proxy issue

### PR DESCRIPTION
**Pull request by rasika-chivate**
Issue wile using reverse proxy

### CODE REVIEW

File: docs/release-notes/2.27.0.md [hunk 1]
```diff
@@ -159,3 +159,19 @@ Simplified experience in registering EKS cluster with kubeconfig generated by ek
...
+
+**Issue**: **Error while using reverse proxy (like Nginx)** [PMM-9992](https://jira.percona.com/browse/PMM-9992)
+
+While using a reverse proxy (for example, Nginx) in front of PMM, you can run into the error `origin not allowed` after upgrading to PMM 2.27.0 or newer versions.
+
+**Solution**
+
+Add the Host header to the reverse proxy config file.
+
+**Example**
```
Comment (nikita-b) [inline docs/release-notes/2.27.0.md:173]: I think we don't need this title.

File: docs/release-notes/2.27.0.md [hunk 2]
```diff
@@ -159,3 +159,19 @@ Simplified experience in registering EKS cluster with kubeconfig generated by ek
...
+
+## Known Issues
+
+**Issue**: **Error while using reverse proxy (like Nginx)** [PMM-9992](https://jira.percona.com/browse/PMM-9992)
+
+While using a reverse proxy (for example, Nginx) in front of PMM, you can run into the error `origin not allowed` after upgrading to PMM 2.27.0 or newer versions.
+
+**Solution**
+
+Add the Host header to the reverse proxy config file.
```
Comment (nikita-b) [inline docs/release-notes/2.27.0.md:171]:
```suggestion
Add the `Host` header to the reverse proxy config file.
```

**Review by nikita-b — APPROVED** [review]

**Review by rnovikovP — APPROVED** [review]

**Comment by rnovikovP** [comment]
There is no need to wait for PMs to add the Known issue for older versions when we already have the text. 
 Also, for such cases, we need Duygu instead of me.

File: docs/release-notes/2.27.0.md (hunk 1 above)
Comment (rasika-chivate) [reply docs/release-notes/2.27.0.md:173]: ok

**Review by denisok — APPROVED** [review]

**Pull request merged**